iphone/Maps/Classes/CustomAlert/BaseAlert/MWMAlert.mm

```diff
@@ -266,4 +266,21 @@ - (void)layoutSubviews {
   [super layoutSubviews];
 }
 
+- (void)traitCollectionDidChange:(UITraitCollection *)previousTraitCollection {
+  [super traitCollectionDidChange:previousTraitCollection];
+  if (self.traitCollection.userInterfaceStyle != previousTraitCollection.userInterfaceStyle) {
+    [self updateViewStyle:self];
+  }
+}
+
+- (void)updateViewStyle:(UIView *)view {
+  if (!view)
+    return;
+  view.isStyleApplied = NO;
```

Comment:
How is this variable used?

Reply:
Uneuseful chunk of code! (copy-pasted from StyleManager).  
thx!